glide.lock

```diff
@@ -40,6 +40,8 @@ imports:
   version: 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
 - name: github.com/JiangtianLi/gettext
   version: a8983c062be4b565d723c478922d7736e04fdba4
+- name: github.com/kelseyhightower/envconfig
```

Comment:
I'm open to any way to add `envconfig` to `glide.yaml`, `glide.lock`, and `vendor/` so long as we don't also deliver unrelated dependency changes (such as deleting `glide.lock` and re-`glide install`'ing would do).